.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Snapshot
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  publish-snapshot:
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        branch:
+          - master
+          - branch-1.3
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@master
+      with:
+        ref: ${{ matrix.branch }}
+    - name: Cache Maven local repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: snapshot-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          snapshot-maven-
+    - name: Install Java 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Publish snapshot
+      env:
+        ASF_USERNAME: ${{ secrets.NEXUS_USER }}
+        ASF_PASSWORD: ${{ secrets.NEXUS_PW }}
+      run: ./build/mvn clean deploy -DskipTests -Pspark-provided -s ./build/release/asf-settings.xml
```

Comment:
`-Pkyuubi-extension-spark-3-1` ?

Reply:
not need to specify `-Papache-release` ?

Reply:
> `-Pkyuubi-extension-spark-3-1` ?

@yaooqinn Should we publish `kyuubi-extension-spark-3-1` to maven central ?

Reply:
> not need to specify `-Papache-release`?

There is no GPG info in GA, I'm planing use `-Papache-release` in RC release.

Reply:
> There is no GPG info in GA, I'm planing use -Papache-release in RC release.

ah I see, thanks